.github/workflows/test.yml

```diff
@@ -19,14 +19,16 @@ jobs:
         uses: actions/cache@v3
         with:
           path: ${{ steps.pnpm-cache.outputs.STORE_PATH }}
-          key: ${{ runner.os }}-${{ matrix.node }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
```

Comment:
There is no `pnpm-lock.yaml` in our repo. `hashFiles('**/pnpm-lock.yaml')` is always `""` (empty string)